projects/demo/src/modules/app/app.providers.ts

```diff
@@ -157,13 +158,14 @@ export const APP_PROVIDERS: Provider[] = [
     {
         provide: TUI_DROPDOWN_HOVER_OPTIONS,
         useFactory: () =>
-            inject(TUI_IS_E2E)
+            inject(TUI_IS_E2E) || inject(TUI_REDUCED_MOTION)
```

Comment:
Should we revert this line (see [this discussion](https://github.com/taiga-family/taiga-ui/pull/5329#discussion_r1319834422)) ?